src/plugins/data_source/server/legacy/http_aws_es/connector.js

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * A connection handler for Amazon ES.
+ *
+ * Uses the aws-sdk to make signed requests to an Amazon ES endpoint.
+ *
+ * @param client {Client} - The Client that this class belongs to
+ * @param config {Object} - Configuration options
+ * @param [config.protocol=http:] {String} - The HTTP protocol that this connection will use, can be set to https:
+ * @class HttpConnector
+ */
+import { Config, Credentials } from 'aws-sdk';
```

Comment:
It seem like that `Config` and `Credentials` were not used directly in the code. We may remove this. it is not block, we can do it later.